src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -33,8 +33,23 @@ public string GetStringValue()
 
             ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
 
+#if !netstandard
```

Comment:
span-based encoding APIs aren't available on netstandard2.0

Reply:
same here, Make sure that the defaults are always to use the netstandard path whenever no constants are defined. That way, we will have the source package not needing to define any constants on the users projects for it to work. In our own project, we can control and define however many constants we want, and that is better than forcing consumers to do so as well.